source/module_io/input_conv.cpp

```diff
@@ -557,45 +557,12 @@ void Input_Conv::Convert()
 //-----------------------------------------------
 // caoyu add for DeePKS
 //-----------------------------------------------
-#ifdef __DEEPKS
-    GlobalV::deepks_scf = PARAM.inp.deepks_scf;
-    GlobalV::deepks_bandgap = PARAM.inp.deepks_bandgap; // QO added for bandgap label 2021-12-15
-    GlobalV::deepks_v_delta = PARAM.inp.deepks_v_delta;
-    GlobalV::deepks_out_labels = PARAM.inp.deepks_out_labels;
-    GlobalV::deepks_equiv = PARAM.inp.deepks_equiv;
-
-    if (GlobalV::deepks_equiv && GlobalV::deepks_bandgap) {
-        ModuleBase::WARNING_QUIT(
-            "Input_conv",
-            "deepks_equiv and deepks_bandgap cannot be used together");
-    }
-    if (PARAM.inp.deepks_out_unittest)
-    {
-        GlobalV::deepks_out_labels = true;
-        GlobalV::deepks_scf = true;
-        if (GlobalV::NPROC > 1)
-        {
-            ModuleBase::WARNING_QUIT("Input_conv", "generate deepks unittest with only 1 processor");
-        }
-        if (PARAM.inp.cal_force != 1)
-        {
-            ModuleBase::WARNING_QUIT("Input_conv", "force is required in generating deepks unittest");
-        }
-        if (GlobalV::CAL_STRESS != 1)
+    #ifndef __DEEPKS
+        if (PARAM.inp.deepks_scf || PARAM.inp.deepks_out_labels || PARAM.inp.deepks_bandgap || PARAM.inp.deepks_v_delta)
```

Comment:
mv it to check_value

Reply:
The parameter has been move to the check_vlaue in PARAM.inp.deepks_scf.